documentation/Simulations getting started.md

```diff
@@ -1,7 +1,7 @@
 # How to design your own processing pipeline
 ## Packages to install (ensure the paths and subfolders are added in Matlab)
-- SimNIBS.
-- k-wave.
+- SimNIBS 4 or higher.
```

Comment:
better not say "or higher" if we don't know whether it would work on newer versions